src/tox_lsr/test_scripts/runqemu.py

```
@@ -85,9 +85,13 @@ def strtobool(val):
 
 def is_ansible_env_var_supported(env_var_name):
     """See if ansible supports the given config env var."""
-    result = subprocess.check_output(  # nosec
-        ["ansible-config", "list"], stderr=subprocess.STDOUT, encoding="utf-8"
-    )
+    try:
+        result = subprocess.check_output(  # nosec
+            ["ansible-config", "list"], stderr=subprocess.STDOUT, encoding="utf-8"
```

Ruff (0.15.20)

[error] Starting a process with a partial executable path

(S607)